Update links to etherscan.io

dstarcev opened the pull request:
- Update links to etherscan.io to reflect recent changes (e.g. separate domain for ropsten)
- Remove duplicated code
- Add some tests

5chdn commented on js/src/3rdparty/etherscan/links.js:
```diff
@@ -15,15 +15,18 @@
+  let prefix = defaultPrefix;
 
   switch (netVersion) {
     case '2':
```
we should remove `'2'` completely from this list, I don't know of any working morden explorer and etherscan recently told me they do not plan to maintain one.

dstarcev commented:
I have removed `morden` and added the recently born `rinkeby`. Parity doesn't support clique yet but when (and if) it does, correct links will already be there.

rphmeier commented:
Doesn't ETC still use morden as a testnet?

gavofyork commented:
yeah i think so.

dstarcev commented:
As @5chdn mentioned, etherscan.io doesn't seem to support morden anymore. https://morden.etherscan.io/ refers to `ROPSTEN (Attacked) TESTNET` and I am not sure if we should provide links to it.

5chdn commented:
> Doesn't ETC still use morden as a testnet?

It's not about removing the morden testnet but the explorer links (as there is none).

dstarcev commented:
I have updated the PR as the mainnet network id is actually `1`.

jacogr commented:
Looks good, tested as well. Just waiting on CLA signoff.

dstarcev commented:
[clabot:check]
